### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3151] [General] [answered]
hey team! quick (?) question: are keyboard shortcuts a requirement from an a11y perspective? i know keyboard nav is, but i wasn't sure about shortcuts. we're working on adding shortcuts to the rich text editor, and wanted to understand more before prioritizing. thank you!

cc: @shannoncawley & @FirstWhack

**Comment by SiTaggart** [comment, marked as answer]
Hi @ceyerkes 

Great question! The way I would approach it is rather than think about accessibility requirements being a set of separate features, think about the user interaction you are trying to achieve, and that should inform you of what you need to do.

So rather than think "is keyboard shortcuts a requirement", think about what the action is. "Is making bold text with just my keyboard a requirement?".

So, if you can create a text selection with your keyboard, navigate out of the RTE to the toolbar, and hit the bold button to make that text selection bold, then you don't need a keyboard shortcut. If you can't do that, say the text selection disappears on navigation or you can't get out of the RTE, then you need a keyboard shortcut.

The question is, can each user perform the same function.

**Reply by ceyerkes** [reply]
Awesome, thank you @SiTaggart!
